locales/en-US/messages.properties

```diff
@@ -197,6 +197,8 @@ navAllowJSTitle=Toggle running JavaScript code in the preview
 navPreviewTitle=View the Preview
 navViewTutorialTitle=View the Project Tutorial
 navDOMElementInspectorIconTitle=Toggle Element Inspector
+navToggleAutoUpdateTitle=Toggle Preview Auto-refresh
+navToggleAutoUpdateLabel=Auto
```

Comment:
Also, I wonder if "Auto" is the right word for this. Might be good to get feedback from non-devs about this once it lands.